mllib/src/main/scala/org/apache/spark/mllib/optimization/GradientDescent.scala

```diff
@@ -77,6 +80,17 @@ class GradientDescent private[mllib] (private var gradient: Gradient, private va
   }
 
   /**
+   * Set the convergence tolerance. Default 0.001
+   * convergenceTol is a condition which decides iteration termination.
+   * If the difference between last loss and last before loss is less than convergenceTol
```

Comment:
Confusing wording.  I would phrase this as "difference between the current loss and the previous loss"